config/site_settings.yml

```diff
@@ -830,6 +830,9 @@ trust:
   min_trust_to_send_messages:
     default: 1
     enum: 'TrustLevelSetting'
+  min_trust_to_send_email_messages:
+    default: 3
```

Comment:
also, lets add one more setting so we can only enable this for staff. default is only staff are allowed this feature.